docs/chapters/licensing/index.rst

```diff
@@ -0,0 +1,25 @@
+Licensing
+=========
+
+Each source code file needs to mention what license applies and who owns
+the copyright in the file header. The purpose of this is to make it
+absolutely clear, for both humans and tooling, which license it is that
+applies to that particular file even when the file is moved to another
+project. In order to achieve this, it has been decided that the header
+should include:
```

Comment:
I agree. The mandate and the terms MUST, MAY, SHOULD, etc. ought to live in the compliance office where there is a consensus driven decision process. The outward facing developer docs ought to be user-centric and explaining our rationale as opposed to setting limits and enforcing our business decisions on devs.